### PR TITLE
Added Config ignore.

### PR DESCRIPTION
**Pull request by sonnykt**
* Added Config Ignore modules.
* The below configs are ignored by default:
```
clamav.settings
login_security.settings
password_policy.*
seckit.settings
system.menu.*
webform.webform.*
webform.webform_options.*
```

GovDex: https://govdex.gov.au/jira/browse/GOVCMSD8-112

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -129,7 +130,12 @@
             "docroot/libraries/{$name}": ["type:drupal-library", "type:bower-asset", "type:npm-asset"],
             "drush/contrib/{$name}": ["type:drupal-drush"]
         },
-        "patches": {}
+        "enable-patching": true,
+        "patches": {
+            "drupal/config_ignore": {
+                "Offset error within IgnoreFilter::activeReadMultiple() - https://www.drupal.org/project/config_ignore/issues/2972302": "https://www.drupal.org/files/issues/2018-06-05/offset-error-within-2972302-6.patch"
```
Comment (gargsuchi) [inline composer.json]: @sonnykt  As per https://www.drupal.org/project/config_ignore/issues/2972302, patch number #13 and not #6 has been merged to dev. Should we not use 
https://www.drupal.org/files/issues/2018-07-31/offset-error-within-2972302-13.patch in that case?

Comment (sonnykt) [reply composer.json]: @gargsuchi you are right, I used an old patch. I updated the PR with the new one.

**Review by gargsuchi — DISMISSED** [review]

**Comment by pandaskii** [comment]
Please resolve the conflict in composer.json

**Review by tobybellwood — APPROVED** [review]
resolved composer.json

**Pull request merged**